.travis.yml

```diff
@@ -1,42 +1,31 @@
 sudo: false
-
 language: python
-
 python:
-  - 3.5
-
+- 3.5
+- 3.8
 env:
-  - TOXENV=django111-drf39 AFTER_SUCCESS=codecov INSTALL_EXTRAS=codecov
-  - TOXENV=django111-drf39
-  - TOXENV=django111-drflatest
-  - TOXENV=django20-drf39
-  - TOXENV=django20-drflatest
-  - TOXENV=django21-drf39
-  - TOXENV=django21-drflatest
-  - TOXENV=django22-drf39
-  - TOXENV=django22-drflatest
-
+- TOXENV=django22-drf39 AFTER_SUCCESS=codecov INSTALL_EXTRAS=codecov
+- TOXENV=django22-drflatest
 matrix:
   include:
-    - python: 3.5
-      env: TOXENV=quality
-    - python: 3.5
-      env: TOXENV=docs
-
+  - python: 3.5
+    env: TOXENV=quality
+  - python: 3.5
+    env: TOXENV=docs
+  - python: 3.8
+    env: TOXENV=quality
```

Comment:
We can remove this from matrix and instread add a new env with TOXENV=quality which should be equivalent to have these two workers for quality test here.

Reply:
that's actually a very good suggestion, thanks